src/client/components/Input/SearchBox.tsx

```diff
@@ -1,5 +1,4 @@
 import styled from 'styled-components';
-import { number } from 'yup';
```

Comment:
I *think* this means that yup is unnecessary now? If so: `npm r`

Reply:
I think this should be in separate PR in case removing it causes issues, might be a peer dep